src-tools/netutils/asio_send_block/asio_send_block.cpp

```diff
@@ -15,15 +90,47 @@ enum { max_length = 655035 };
 
 using mysize_t = uint64_t; // size_t is too small to count some things, and overflows, e.g. on r-pi
 
+template <class TOut>
+void explode(const std::string & text, char sep, TOut & output) {
+    std::istringstream iss(text);
+    std::string word;
+    while (std::getline(iss, word, sep)) output.push_back(word);
+}
+
+std::vector<std::string> explode(const std::string & text, char sep) {
+	std::vector<std::string> ret;
+	explode(text,sep,ret);
+	return ret;
+}
+
+std::string remove_extra_whitespace(const std::string  & str) {
```

Comment:
Erase–remove idiom

Reply:
not so trivial to do with just like
v.erase( std::remove( v.begin(), v.end(), 5 ), v.end() );

because the test does not test given element only, 
it tests given element + it's preceding, or element + global state

imo easier to do it like that (or propose your version of it)